backend/src/middlewares/errorHandler.ts

```diff
@@ -16,20 +20,34 @@ export const errorHandler = (
   next: NextFunction,
 ) => {
   if (error.name === "ValidationError") {
-    return res.status(400).send({
+    return res.status(400).json({
       type: "ValidationError",
-      details: error.details,
+      errorMessage: error.details[0].message,
     });
-  }
-
-  if (error instanceof CustomError) {
+  } else if (error.name === "NotFoundError") {
```

Comment:
This error handler is nice but as the project grows the if/else statements will grow too and will be impossible to read in some time. My suggestion is to split this in multiple smaller methods that check different things smth like this handleErrorName(error.name); handleErroCode(error.code); handleErrorKind(error.kind) and then in the implementation of those you can use a switch statement instead of an if else statement. That would simplify a lot the code and will make more maintainable